sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -117,7 +117,7 @@
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "2.0.0-beta.7",
```

Comment:
We should not be pinning the dependency anymore once it is out of beta. Please use ^